### PR TITLE
SEO-171911 - WPF Control missing h1

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Please review the changes for missing H1 below.

Regards,
Sharly A.

### CODE REVIEW

File: wpf/Release-notes/Migratingtov16.1.0.24.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ platform: wpf
 documentation: ug
 keywords: migration, upgrade-changes, 2018vol1-changes
 ---
-## SfRichTextBoxAdv
+# Volume 1 Migration document in WPF RichTextBox(SfRichTextBoxAdv)
```
Comment (gowthamanthavasiyappan) [inline wpf/Release-notes/Migratingtov16.1.0.24.md:8]: Check control name properly.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: wpf/Release-notes/Migratingtov16.1.0.24.md (hunk 1 above)
Comment (Sharly-Atieno) [reply wpf/Release-notes/Migratingtov16.1.0.24.md]: done

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: wpf/Release-notes/Migratingtov16.1.0.24.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-title: Essential Studio for WPF 2018 volume 1 Migration document
-description: Essential Studio for WPF 2018 volume 1 Migration document
+title: Volume 1 Migration Document in WPF Release Notes | Syncfusion
+description: Learn about volume 1 migration document support in Syncfusion Essential WPF Release Notes, its elements and more.
 platform: wpf
 documentation: ug
 keywords: migration, upgrade-changes, 2018vol1-changes
 ---
-## SfRichTextBoxAdv
+# Volume 1 Migration document in WPF Release Notes
```
Comment (Girija-Rajendran) [inline wpf/Release-notes/Migratingtov16.1.0.24.md:8]: Please check title, description, and H1

**Pull request closed without merging**